src/layouts/ScrollToTop/ScrollToTop.module.css

```diff
@@ -0,0 +1,28 @@
+.top-to-btn{
+  position: relative;
+}
+
+.icon-position{
```

Comment:
I would merge .icon-position and .icon-style in a single class and call it just .icon, so that then the below modifiers specified with -- will apply to the .icon block ( see BEM specification for the naming rules )

Reply:
https://getbem.com/naming/